### DISCUSSION TITLE
What is the current proposal in use by Mac OS Montery kept up to date ?

### DISCUSSION THREAD

**Discussion by daniel-dlds** [gitbls/pistrong#16] [Q&A]
I had the connection to my home vpn working some time ago. I had to add some values to the proposal properies in the file 👍 pistrong-CAServerConnection.conf :
`proposals = aes256-sha256-modp2048, aes256-sha256-modp1024,aes256-sha1-modp1024,aes256-sha256-prfsha256-modp2048`
Now I can't connect again. Does anyone know if the cypher has changed again ?
All I get in the logs is :
`Jul 28 15:27:51 vpnserver charon-systemd[603]: XXX.XXX.XXX.XXX is initiating an IKE_SA
Jul 28 15:27:51 vpnserver charon-systemd[603]: selected proposal: IKE:AES_CBC_256/HMAC_SHA2_256_128/PRF_HMAC_SHA2_256/MODP_2048
Jul 28 15:28:21 vpnserver charon-systemd[603]: deleting half open IKE_SA with 85.244.10.245 after timeout`

**Comment by gitbls** [comment]
If the ciphers had changed, I'd expect to see a _no acceptable proposal found_ error, rather than a timeout. I just reviewed the open StrongSwan MacOS-related issues, but didn't see anything since the change requiring `aes256-sha256-prfsha256-modp2048`, which you have. 

I don't have access to a MacOS system to test this myself, but I'd start with rebooting the server, check for errors, and try to connect again.

If you'd like my assistance in chasing this down, please run `pscollect` from this GitHub and create an issue.

**Reply by daniel-dlds** [reply]
I was connecting to the VPN using my internet domain name for the vpn, while inside the local network. I remember testing the vpn connection this way and it used to work.
Now I am outside my LAN away from home, and it is working. So please ignore this thread. All is working as it's supposed.
Thanks for your help
